gc/base/ModronAssertions.h

```diff
@@ -30,6 +30,9 @@
 #include "omrcfg.h"
 #include "omrcomp.h"
 #include "omr.h"
+#undef _UTE_MODULE_HEADER_
+#undef UT_MODULE_LOADED
+#undef UT_MODULE_UNLOADED
```

Comment:
@RSalman I am not sure about these lines. But I have to add them to remove redefinition errors.

Reply:
This shouldn't be necessary, it doesn't look right. I'm not sure why, but there must be an underlying def issue in the changes.

Reply:
@RSalman 
I agree this is one aggressive approach; it may jeopardize the the point to have these variables as guards. 
The definition issue arises from `VerboseHandlerJava.cpp` which includes `VerboseBuffer.hpp` in recent changes.
`VerboseHandlerJava.cpp=>VerboseBuffer.hpp=>ut_j9vgc`
`VerboseHandlerJava.cpp=>FinalizeListManager=>ut_j9mm`

Both `ut_j9vgc` and `ut_j9mm` define above variables, thus the redefinition. Since the issue is in the `VerboseHandlerJava.cpp`, they should be undefined in `VerboseHandlerJava.cpp` as well. Please check my new commit.

https://github.com/eclipse/omr/pull/5607/commits/6cb15bd9b4817c0e7d1943b8cf11b0fc3c290e04
https://github.com/eclipse/openj9/pull/10879/commits/15c8bbf7c36ce20ab639701a1fb7e39dd86c582e

Reply:
ah okay, that makes sense. It looks okay and it should be fine, although, I don't know if it's "the right approach" or if we should work around it. The hesitation comes from reading existing comments saying _" OMRTODO temporary workaround to allow .... "_ in some places were something similar  is done. Leave it for now and I'll try to take a look.

Reply:
> VerboseHandlerJava.cpp=>FinalizeListManager=>ut_j9mm

I don't see `ut_j9mm` being included in `FinalizeListManager.hpp`, am I missing something?

Also, if you haven't already, try switching order of includes. `VerboseBuffer.hpp` already undef these, try putting `VerboseBuffer.hpp` last, after `FinalizeListManager`.

Reply:
My Bad, the complete sequence should be:
`VerboseHandlerJava.cpp=>FinalizeListManager=>ObjectAccessBarrier=>ModronAssertions=>ut_j9mm`
```
In file included from /root/code/openj9-openjdk-jdk11/omr/gc/base/ModronAssertions.h:33:0,
                 from /root/code/openj9-openjdk-jdk11/openj9/runtime/gc_base/ObjectAccessBarrier.hpp:41,
                 from /root/code/openj9-openjdk-jdk11/openj9/runtime/gc_base/FinalizeListManager.hpp:38,
                 from /root/code/openj9-openjdk-jdk11/openj9/runtime/gc_verbose_java/VerboseHandlerJava.cpp:36:
/root/code/openj9-openjdk-jdk11/build/linux-x86_64-normal-server-release/vm/runtime/omr/gc/ut_j9mm.h:30:0: error: "UT_MODULE_LOADED" redefined [-Werror]
```
And move app around does work! Thanks.